MAVProxy/modules/mavproxy_adsb.py

```diff
@@ -80,7 +81,10 @@ def __init__(self, mpstate):
                                                      ("show_threat_radius", bool, False),
                                                      # threat_radius_clear = threat_radius*threat_radius_clear_multiplier
                                                      ("threat_radius_clear_multiplier", int, 2),
-                                                     ("show_threat_radius_clear", bool, False)])
+                                                     ("show_threat_radius_clear", bool, False),
+                                                     ("alt_color1", str, "blue"),
+                                                     ("alt_color2", str, "red"),
```

Comment:
Could have been `alt_color_close` and `alt_colour_far`

... and `alt_colour_close_thresh`